interview/password.py

```diff
@@ -15,5 +15,27 @@ def generate_password() -> str:
     # https://docs.python.org/3/library/random.html#random.randint
 
     # TODO implement generate_password function
+    import random
+
+    password = ''
+
+    alphabet ='abcdefghijklmonpqrstuvwxyz'
+    r = alphabet[random.randint(0, len(alphabet)-1)]
```

Comment:
not bad -- but the issue with these passwords is that they all would start with duplicate letters when using random this way (ex: 'aaaaa' ...)